spec/models/labware_creators/common_file_handling/csv_file/row_base_spec.rb

```diff
@@ -47,6 +47,15 @@
         )
       end
     end
+
+    context 'when row_data contains valid characters and empty cells' do
```

Comment:
maybe have a test for a cell with spaces in it '    ' as well as the empty string one